web/static/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html><html lang="en"><head>
+  <meta charset="utf-8">
+  <title>Home Edge</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+<body>
+Home Edge Web UI
+</body></html>
```

Comment:
Improper indentation followed, somewhere there are spaces and newlines, in some there isn't, need to fix

Reply:
Done